From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: fukiame <fukiame@proton.me>
Date: Tue, 4 Jun 2024 14:50:24 +0530
Subject: soong: add vendor/niigo-priv/keys to allowlist

inspired by https://github.com/LineageOS/android_build_soong/commit/7d4531838a6a1bd0d4d47d48080d83786510ad98

add our entry to the middle of the list to make unattended patching easier
(no roms would do retartded thing like modifying this part, they add to the end of the list instead)

Signed-off-by: Sushrut1101 <guptasushrut@gmail.com>

diff --git a/android/allowlists/allowlists.go b/android/allowlists/allowlists.go
index 6e22831..969ad76 100644
--- a/android/allowlists/allowlists.go
+++ b/android/allowlists/allowlists.go
@@ -534,6 +534,9 @@ var (
 		"prebuilts/runtime":/* recursive = */ false,
 		"prebuilts/rust":/* recursive = */ true,
 
+		// T25 - throw our key into soong allowlist
+		"vendor/niigo-priv/keys":/* recursive = */ false,
+
 		// not recursive due to conflicting workspace paths in tools/atest/bazel/rules
 		"tools/asuite/atest":/* recursive = */ false,
 		"tools/asuite/atest/bazel/reporter":/* recursive = */ true,
-- 
2.44.0

